### PR TITLE
Pass of paper

### PR DESCRIPTION
**Pull request by FedericoAureliano**
1. I was wrong about the capitals in figures and tables. Fixed.
2. Also bunch of other stuff

### CODE REVIEW

File: includes/abstract.tex [hunk 1]
```diff
@@ -1,10 +1,10 @@
 \begin{abstract}
     In this paper, we introduce \fuzzer{}: a modular \smt{} problem instance
-    transformer and generator for string solvers. We supply a
```
Comment (dblotsky) [inline includes/abstract.tex]: The "for string solvers" part was initially conceived by Vijay, and I think it's worth it to keep it. Do you feel it's better without it?

File: includes/introduction.tex [hunk 2]
```diff
@@ -8,30 +8,30 @@ \section{Introduction}
...
+fixed benchmarks remain stagnant, leaving increasing functionality untested.
+As such, there is an acute need for a more robust, inexpensive, and automatic way
 of generating benchmarks to test
 the correctness and performance of SMT solvers.
 
-Fuzzing is often used to test all kinds of software,
-including SAT solvers~\cite{fuzzsat}. Inspired by the utility of fuzzers,
-we introduce a fuzzer for string solvers, \fuzzer{}, and describe its value
+Fuzzing has been used to test all kinds of software
+including SAT solvers~\cite{fuzzsat}. Inspired by these fuzzers,
```
Comment (dblotsky) [inline includes/introduction.tex]: I think Vijay added "the utility of". Would you be ok with keeping it?

File: includes/introduction.tex [hunk 3]
```diff
@@ -8,30 +8,30 @@ \section{Introduction}
...
+we introduce \fuzzer{} and describe its value
 as an exploratory testing tool. We demonstrate its efficacy
-by presenting limitations it helped us find in
+by presenting limitations it helped discover in
 leading string solvers. To the best of our knowledge, \fuzzer{} is the
-first tool aimed at automatic generation of string constraints. \fuzzer{} can
+only tool aimed at automatic generation of string constraints. \fuzzer{} can
```
Comment (dblotsky) [inline includes/introduction.tex:23]: Just curious: why "first" to "only"? "First" implies "only", but remains true as other ones are developed.

File: includes/stringfuzz.tex [hunk 4]
```diff
@@ -5,110 +5,119 @@ \subsubsection{Implementation and Architecture}
...
+    This tool takes an \smt{}
+    instance as input and outputs its properties: the number of
+    variables/literals, the max/median syntactic depth of expressions, the
+    max/median literal length, etc.
 \end{description}
 
+We organized \fuzzer{} to be easily extended. As evidence of our success we note
+that while the whole project 
+contains \linesInFuzzer{} lines of code, it takes an average of 
+\linesPerX{} lines of code to create a transformer. \fuzzer{} can either be
```
Comment (dblotsky) [inline includes/stringfuzz.tex]: Maybe change "while the whole project contains ... a transformer." to "the whole project contains ... of code, but transformers contain only \linesPerX{} lines on average."

File: includes/stringfuzz.tex [hunk 5]
```diff
@@ -5,110 +5,119 @@ \subsubsection{Implementation and Architecture}
...
     \centering
-    \caption{\generator{} generators.}
+    \caption{\generator{} built-in generators.}
     \label{tbl:generators}
     \begin{tabular}{|l|l|}
         \hline
         \textbf{Name}
-            & \textbf{Generated instances have ...}
-            \\ \hline
+        & \textbf{Generate instances that have ...}
```
Comment (dblotsky) [inline includes/stringfuzz.tex]: Maybe make this "Generates instances that have ...".

**Review by dblotsky — APPROVED** [review]

**Comment by dblotsky** [comment]
@FedericoAureliano I've added my comments. Feel free to merge after.

File: includes/introduction.tex (hunk 3 above)
Comment (FedericoAureliano) [reply includes/introduction.tex:23]: First ages better, only is stronger for today (when we'll be reviewed).

**Comment by FedericoAureliano** [comment]
@dblotsky I've made all the changes I want. Do you still approve?

**Pull request merged**